### PR TITLE
Only members authentication

### PR DESCRIPTION
**Pull request by bertil291utn**
### Hey guys, we'd appreciate your feedback and suggestions.
We have achieved these activities according to [Members-Only assignment](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication#project-2-members-only):
- Used DEVISE gem for authentication
- Create a model **Admin** (username, email, and password) with Devise CLI `rails g model devise <name>`
- Add a **Post** (title and body ) model (Model, controller, and views), and link to **Admin** model through `admin_id` field to know which **Admin** wrote a **Post**.
- Only members who have logged in got access to see which one has written a post
- When the member is not logged in they just see list a post with no username
Thank you for your time 💻 🚀

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
## Hi team ! Very nice project! :wave: 

# You did a great job :clap:  a few changes before it can be approved :eyes:  

## [OPTIONAL] 
* What about adding front-end to your project? You don't need to be a designer, but you finished HTML/CSS part, and already know CSS, SCSS, Bootstrap you can use one!

## [REQUIRED] 
#### Posts


* [x] ROUTES: It creates the application routes using `resources :posts, only: [:new, :create, :index]`
screen

* [x] PostsController :  it uses `before_action` to restrict access to the `:new` and `:create` methods (uses ApplicationController helper method), from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication)
screenshot 

new



### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding * <g-emoji class="g-emoji" alias="computer" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f4bb.png">computer</g-emoji>

**Comment by bertil291utn** [comment]
### Thank you @Anna-Myzukina for code review.

**Required request**

It was already working though but we didn't achieve as assignment requirement and in the end we fix them.

**Optional request**

We are using Bootstrap and some CSS in `aplication.css` and `scaffold.scss`.

Thank you for your time 💻 🚀

**Comment by bertil291utn** [comment]
# Thank you @Anna-Myzukina for code review.
## Required request

It was already working though but we didn't achieve as assignment requirement and in the end, we fix them.

## Optional request

We are using Bootstrap and some CSS in `aplication.css` and `scaffold.scss`.
<br>
<hr>
Thank you for your time 💻 🚀

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+end
```
Comment (Torres-ssf) [inline app/models/user.rb:6]: Is there a reason why you have both User and admin models? :thinking:

Comment (bertil291utn) [reply app/models/user.rb:6]: **Thanks for code review.**
We were using User to test the project but we forgot to deleted it. So now it's gone 
Thank you 💻  🚀

**Comment by bertil291utn** [comment]
# Thank you @Torres-ssf for code review
## Required request

- The issue about routing after a **Post** is created is [solved ](https://github.com/bertil291utn/members-only/pull/1/commits/0a1b8931a580c13beca5c4d593179ae70f883358)✔️ 
- And about your question we are using **Admin** instead of **User**, we just forgot to delete that User scaffold, also [solved](https://github.com/bertil291utn/members-only/pull/1/commits/0a1b8931a580c13beca5c4d593179ae70f883358) ✔️ 

<br>
<hr>
Thank you for your time 💻 🚀

**Pull request closed without merging**